adb_graphics/datahandler/gribdata.py

```diff
@@ -473,7 +479,9 @@ def aviation_flight_rules(self, values, **kwargs) -> np.ndarray:
         flru = np.where((ceil > 0.0) & (ceil < 0.5), 3.01, flru)
         flru = np.where((vis < 1.), 3.01, flru)
 
-        return flru
+        vis.close()
+
+        return xr.DataArray(flru)
```

Comment:
The numpy "where" statements above returned a numpy array. Just making sure here that we do get an XArray